docs/metrics/exemplars/README.md

```diff
@@ -1,99 +1,154 @@
 # Using Exemplars in OpenTelemetry .NET
 
+- [Install and run Jaeger](#install-and-run-jaeger)
```

Comment:
nit: Seems we do this other places but IMO we should have a heading here like "Table of contents"

Reply:
Added.